core/src/consensus/sortition/binom_cdf.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use statrs::distribution::{Binomial, Univariate};
+
+/// Determine in which the lottery value is included walking along the
+/// binomial cumulative distribution function.
+///
+/// # Arguments
+///
+/// * `p`   - probability parameter in the binomial distribution
+/// * `n`   - trial prameter in the binomial distribution
+/// * `lottery` - the target lottery value in the closed interval [0, 1]
+/// * `voting_power` - determines the number of walks in cdf, the maximum possible number of walks is n in the binomial distribution
```

Comment:
determines the number of walks in cdf, where the maximum possible number of walks is n in the binomial distribution

Reply:
fixed